components/tyk-mdcb/templates/deployment-mdcb.yaml

```
@@ -0,0 +1,144 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mdcb-{{ include "tyk-mdcb.fullname" . }}
+  labels:
+    {{- include "tyk-mdcb.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.mdcb.autoscaling.enabled }}
+  replicas: {{ .Values.mdcb.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "tyk-mdcb.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.mdcb.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "tyk-mdcb.labels" . | nindent 8 }}
+        {{- with .Values.mdcb.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.mdcb.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "tyk-mdcb.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.mdcb.podSecurityContext | nindent 8 }}
+      containers:
+        - name: mdcb-{{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.mdcb.containerSecurityContext | nindent 12 }}
+          image: "{{ .Values.mdcb.image.repository }}:{{ .Values.mdcb.image.tag }}"
+          imagePullPolicy: {{ .Values.mdcb.image.pullPolicy }}
+          command: ["/opt/tyk-sink/tyk-sink", "--c=/etc/tyk-sink/tyk_sink.conf"]
+          ports:
+            - containerPort: {{ .Values.mdcb.listenPort }}
+            - containerPort: {{ .Values.mdcb.probes.healthCheckPort }}
+          livenessProbe:
+            httpGet:
+              path: {{ .Values.mdcb.probes.liveness.path }}
+              port: {{ .Values.mdcb.probes.healthCheckPort}}
+              scheme: {{ .Values.mdcb.probes.liveness.scheme }}
+            initialDelaySeconds: {{ .Values.mdcb.probes.liveness.initialDelaySeconds }}
+            periodSeconds: {{ .Values.mdcb.probes.liveness.periodSeconds }}
+            timeoutSeconds: {{ .Values.mdcb.probes.liveness.timeoutSeconds }}
+            failureThreshold: {{ .Values.mdcb.probes.liveness.failureThreshold }}
+          readinessProbe:
+            initialDelaySeconds: {{ .Values.mdcb.probes.readiness.initialDelaySeconds }}
+            periodSeconds: {{ .Values.mdcb.probes.readiness.periodSeconds }}
+            timeoutSeconds: {{ .Values.mdcb.probes.readiness.timeoutSeconds }}
+            failureThreshold: {{ .Values.mdcb.probes.readiness.failureThreshold }}
+            httpGet:
+              path: {{ .Values.mdcb.probes.readiness.path }}
+              port: {{ .Values.mdcb.probes.healthCheckPort}}
+              scheme: {{ .Values.mdcb.probes.liveness.scheme }}
```

```suggestion
              scheme: {{ .Values.mdcb.probes.readiness.scheme }}
```
Can't we configure scheme ourselves like we do in other components?